modules/special-rules/pages/super-lifta-droppa.adoc

```diff
@@ -3,18 +3,36 @@
 This unusual but powerful weapon is unique to the Orks.
 Human tech-priests are at a loss to explain how it works, except to note its effects.
 
-Ork Mekboys use the weapon to focus a traktor beam on the target and then lift the target high above the ground -- or above a juicy secondary target nearby.
-Then they drop the first target onto the secondary target...
+Ork Mekboys use the weapon to focus a traktor beam on the target and then lift the target high above the ground -- or above juicy secondary targets nearby.
+Then they drop the first target onto the secondary targets...
 
 ---
 
-When you shoot with a {super-lifta-droppa}:
+When you make a {super-lifta-droppa} shot:
 
-. You can target Vehicle and War Engine units, but not Infantry units.
-. To hit and lift the target unit, roll a D6: You need to score higher than the initial Damage Capacity value of the target unit.
-(Vehicle units have a Damage Capacity value of 1; War Engine units vary, so check their data sheet.)
-. For each point by which your score is higher than the target unit's initial Damage Capacity value, you can move it up to 1 cm in any direction to determine where the {super-lifta-droppa} will drop it.
-. The target unit is dropped at the spot you chose, and is automatically destroyed -- if it is a War Engine unit, roll on its Catastrophic Damage table immediately.
-. If there is a secondary target at the spot where the target unit is dropped, roll a D6: On a result of 4+ the secondary target takes a hit.
+. Pick any Vehicle or War Engine target within range and line of sight, even a specific unit in a detachment.
+You cannot target Infantry units.
+. Roll a D6.
+If you roll a result that is higher than the initial Damage Capacity value of the target unit, then the weapon hits.
++
+Vehicle units usually have a Damage Capacity value of 1.
+War Engine units vary, so check the target unit's data sheet.
+. If the target unit has the {save} ability or the {holo-fields} ability, then your opponent may attempt the save.
+If they are successful then the target is unharmed and nothing further happens -- do not continue below.
+. For each point by which your result is higher than the target unit's initial Damage Capacity value, you can move the unit up to 1 cm in any direction, above other game pieces.
+This is where the {super-lifta-droppa} will drop the target unit.
+. Resolve the drop:
+* The target unit is automatically destroyed.
+If it is a War Engine unit, roll for a result on its Catastrophic Damage table.
+* For each unit that is under the target unit, roll a D6.
+On a roll of 4+, that unit suffers a hit.
++
+{blank}[TODO: How do we define 'under', should we reference some of the same points as for the Barrage template.]
```

Comment:
Copied from the 'outdated' conversation:

Okay, we could add something to the effect of:

>NOTE: In this case, we define 'under' as any part of the unit’s base -- no matter the unit type -- is under the dropped unit or touched by the edge of the outline of the dropped unit.

...but I really don't like the way that this breaks with the conditions we state at `barrage-templates/adoc`.
I get that the weapon might be seen as a bit weak, but I don't think that we should try to offset this by making infantry arbitrarily easier to hit than is usual for the otherwise similar template rules, just for this special case.
Such an odd break in the continuity feels like bad practice and needlessly complex to me.

[So, in lieu of some other bright idea, I think we ought to stick with the 'under' definitions we have for Barrage templates.]

Reply:
I understand where you're coming from in terms of consistency, but from this end it would feel like an arbitrary rules change and I suspect it has implications you're not expecting. And that's ignoring the balance problems, and that the only feedback we have from players was very strongly in favour of hitting everything.

The big issue is that the dropped unit (obviously!) isn't a Barrage template.

The end result is that we massively increase the difference between hitting vehicle units and infantry units, and the smaller the dropped unit, the _bigger_ the difference. Which results in an even bigger difference between based and unbased vehicle models!

Rephrased, the definition for 'under' that's used for Barrage template weapons only 'works' as a rule because the template is substantially larger than the infantry bases. As soon as the 'template' gets smaller than the bases, it stops working well as a rule.

I'm on the road most of this week, but I can try and do some diagrams showing how the 'under' definition doesn't scale well.

Reply:
Back again!

Part of the problem is that I'm not a fan of treating infantry and vehicle units differently in the first place, and I still haven't worked out what GW intended in using it for Barrages in the first place. Especially given how abstract Barrages are, where you're not even shooting at the units under the template.

The next problem is that the text that says 'cover at least three trooper models for an infantry unit' is specific to the Barrage SHW and is never mentioned for _any_ other weapon, even in the FAQs. I'm not 100% sure it's supposed to apply to Mega Cannon or Vortex Missiles!

If we were doing a complete new edition we could treat everything the same and say that a unit is 'under' if any part of its base is under/touching the template. But that has some pretty big implications for internal balance, so I don't want to touch that one just yet. ;-)

> However, as I say, we could insert something like the note I suggested above. I mentioned my distaste and reasoning in connection with it — but we can apply this suggestion (or similar) if you think that's the lesser evil. :-) At least then it is defined rather than left hanging and ambiguous.

Aye.

Reply:
Welcome back!

>Part of the problem is that I'm not a fan of treating infantry and vehicle units differently in the first place, and I still haven't worked out what GW intended in using it for Barrages in the first place. Especially given how abstract Barrages are, where you're not even shooting at the units under the template.

Yes, I agree entirely. It's just that an inconsistent rule is bad enough — I don't like to add it with inconsistent application of the inconsistent rule. :-D Why GW saw fit to differentiate between infantry and the rest in this way I have no idea — it does  seem a bit daft. But so long as we do apply it, there's always a way to rationalise it (infantry have more situational awareness, can scurry out of the way or take cover, etcetera).

>The next problem is that the text that says 'cover at least three trooper models for an infantry unit' is specific to the Barrage SHW and is never mentioned for any other weapon, even in the FAQs. I'm not 100% sure it's supposed to apply to Mega Cannon or Vortex Missiles!

It's true that they only stated the 'cover three models on an infantry stand' requirement in the Barrage topic (as far as I can recall anyway) . That doesn't mean that it was meant to be 'specific to Barrage SHW' — just that they never clarified whether or not we should extend the same logic to other weapons. So I take the view that, as they offered no other definition of 'under' or 'covered by' in any other context, we should assume consistent application of the one definition that we do actually have.

I'd be quite surprised if the designers didn't apply the same requirement rule for other weapons, especially given that they use the same template. It's not impossible that they had a different interpretation for other weapons and never bothered to mention it, but I think this is much less likely. Occam's Razor again, really.

>If we were doing a complete new edition we could treat everything the same and say that a unit is 'under' if any part of its base is under/touching the template. But that has some pretty big implications for internal balance, so I don't want to touch that one just yet. ;-)

Indeed. Something to consider at some point perhaps, but not to rush into. Fair enough.

Okay, I'll hold my nose and apply this:
```suggestion
NOTE: In this case, we consider a unit to be 'under' where any part of the unit’s base is under or touched by the outline of the target unit -- no matter the unit type.
```